.travis.yml

```diff
@@ -1,14 +1,20 @@
 language: node_js
+sudo: false
```

Comment:
sudo is already defined below. I think you should remove this one.

Reply:
done